ConcernsCaseWork/ConcernsCaseWork.Tests/Mappers/NtiMappingTests.cs

```diff
@@ -17,19 +17,25 @@ public class NtiMappingTests
 	{
 		[Test]
 		public void WhenMapDtoToServiceModel_ReturnsCorrectModel()
-		{
-			//arrange
+		{
+			//arrange
+			var now = DateTime.Now;
+
 			var testData = new
 			{
 				Id = 1L,
 				CaseUrn = 123,
 				CreatedAt = DateTime.Now.AddDays(-5),
-				ClosedAt = DateTime.Now,
+				ClosedAt = now,
```

Comment:
Not a blocker, but I'd be tempted to put a different date for each of the ClosedAt, UpdatedAt, DateNTIClosed. Otherwise you can't be sure if they are mapped correctly